two_factor_otp_auth/data/ir_actions_server_data.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="1">
```

Comment:
since version `12.0` `<data>` tag is redundant - you can use `<odoo noupdate="1">` instead

Reply:
removed